doc/source/docs/user_guide/fiona_to_pyogrio.rst

```diff
@@ -0,0 +1,69 @@
+.. currentmodule:: geopandas
+
+.. ipython:: python
+   :suppress:
+
+   import geopandas
+
+Migration from the Fiona to the Pyogrio IO engine
+=================================================
+
+Since version 0.11, GeoPandas supports two IO engines to read and write files:
```

Comment:
Can we generally avoid using "IO" in the text? It is clear to us but may not be to users.